css/syntaxes.json

```diff
@@ -27,7 +27,7 @@
   "clip-source": "<url>",
   "color": "<rgb()> | <rgba()> | <hsl()> | <hsla()> | <hex-color> | <named-color> | currentcolor | <deprecated-system-color>",
   "color-stop": "<color> <length-percentage>?",
-  "color-stop-list": "<color-stop>{2,}",
+  "color-stop-list": "<color-stop>#{2,}",
```

Comment:
It's a mistake in css-image-3, because color-stop-list with no comma doesn't work in browsers and all examples in spec contain a comma.
I filled [an issue](https://github.com/w3c/csswg-drafts/issues/458) about mistake about half year ago, but no reaction yet.